src/plugins/CoreLoadPlugin.ts

```diff
@@ -83,7 +83,17 @@ export default class DojoLoadPlugin {
 			compilation.moduleTemplate.plugin('module', (source: any, module: any) => {
 				if (module.meta && module.meta.isPotentialLoad) {
 					const path = stripPath(basePath, module.userRequest);
-					const require = `var require = function () { return '${path}'; };`;
+					const require = `var require = (function () {
+						var globalScope = typeof window === 'undefined' ? global : window;
```

Comment:
We do seem to have several ways in Dojo 2 of getting a reference to the global, whether it is https://github.com/dojo/shim/blob/79882ec5c4fa78eb91171e7ff42ace275aa68562/src/support/global.ts or https://github.com/dojo/loader/blob/master/src/loader.ts#L8 . Maybe we want to normalize this?